static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -143,7 +143,8 @@ define([
                     && !config.page.isAdvertisementFeature
                     && config.page.pageId !== 'offline-page'
                     && !config.page.shouldHideAdverts
-                    && config.page.section !== 'childrens-books-site') {
+                    && config.page.section !== 'childrens-books-site'
+                    && !config.tests.abNewHeaderVariant) {
```

Comment:
The current ad banner sticks until the bottom of the nav, but with the newly designed slim header on desktop, we would be massively reducing viewability.

I think @stephanfowler suggested we could work around this by sticking the ad banner + nav until an arbitrary scroll position, e.g. 500px. Definitely something we need to think about before it goes into production!